install/install.sh

```diff
@@ -39,6 +33,10 @@ usage() {
 
 
     Extra Variables:
+    * Specify the version of the quipucords server to install.   (defaults to the latest release):
```

Comment:
Remove the awkward spacing before `(defaults...`

Reply:
Done